data/conferences/2017/2017-06-08-jsheroes-2017.yaml

```diff
@@ -18,5 +18,310 @@
     country: Romania
     city: Cluj
   description: |-
-
 talks:
+
+  - title: Unleash the power of Angular Reactive Forms
+    lang: en
+    type: regular
+    time: 2017-06-08
+    room:
+    authors:
+      - name: Nir Kaufman
+        twitter:
+        github:
+        website:
+    slides:
+      -
+    videos:
+      - https://youtu.be/9NXVQsQ8wns
+    description: |-
```

Comment:
The talk descriptions from YT video descriptions are missing, e.g. https://www.youtube.com/watch?v=9NXVQsQ8wns&list=PLB9NqTp0uKrQMLMSeh8RIobwjo5eABnmp&index=4

Reply:
okay let me add it

Reply:
okay I'll upload tonight after adding descs